modules/discretedata/connectivity/cell.h

```diff
@@ -35,40 +35,40 @@ namespace discretedata {
 /** Cell types of vtk **/
 enum class CellType {
     // Linear cells
-    EMPTY_CELL = 0,
-    VERTEX = 1,
-    POLY_VERTEX = 2,
-    LINE = 3,
-    POLY_LINE = 4,
-    TRIANGLE = 5,
-    TRIANGLE_STRIP = 6,
-    POLYGON = 7,
-    PIXEL = 8,
-    QUAD = 9,
-    TETRA = 10,
-    VOXEL = 11,
-    HEXAHEDRON = 12,
-    WEDGE = 13,
-    PYRAMID = 14,
+    EMPTY_CELL       = 0,
```

Comment:
See comment above...
Unrelated to this we really should not use capital letter here. We _only_ use capital letters for MACROS

Reply:
Isn't that already in? I mean this should not be part of the code review. I just pops up because of the format change.

Reply:
Yes, I can change this...